Cargo.toml

```diff
@@ -2,6 +2,7 @@
 
 members = [
     "applehat",
+    "btsensor",
```

Comment:
might be better to just name-grab bthome and move the atc parsing somewhere else?